src/load-app.ts

```diff
@@ -25,12 +25,12 @@ export async function loadApp(appPath: string = "."): Promise<App> {
     // Read the app's exported definition
     let [definition, definitionPath] = await readDefinition<AppDefinition>(appPath, ".", "ShipEngine Integration Platform app");
 
-    if ("deliveryServices" in definition) {
-      let pojo = await readCarrierAppDefinition(definition, definitionPath, manifest);
+    if (isCarrierApp(definition)) {
+      let pojo = await readCarrierAppDefinition(definition as CarrierAppDefinition, definitionPath, manifest);
       return new sdk.CarrierApp(pojo);
     }
     else {
-      let pojo = await readOrderAppDefinition(definition, definitionPath, manifest);
+      let pojo = await readOrderAppDefinition(definition as OrderAppDefinition, definitionPath, manifest);
```

Comment:
```suggestion
      let pojo = await readOrderAppDefinition(definition, definitionPath, manifest);
```

Casting the `definition` isn't necessary because I changed `isCarrierApp` to a type guard